src/main/java/bio/terra/cloudfiletodatastore/deltalayer/functions/DeltaLayerCloudFunction.java

```diff
@@ -0,0 +1,27 @@
+package bio.terra.cloudfiletodatastore.deltalayer.functions;
+
+import bio.terra.cloudfiletodatastore.FileCreatedMessageHarness;
+import bio.terra.cloudfiletodatastore.FileMessage;
+import bio.terra.cloudfiletodatastore.GsonConverter;
+import bio.terra.cloudfiletodatastore.deltalayer.DeltaLayerMessageProcessor;
+import com.google.cloud.functions.CloudEventsFunction;
+import com.google.events.cloud.storage.v1.StorageObjectData;
+import io.cloudevents.CloudEvent;
+import java.util.Objects;
+
+public class DeltaLayerCloudFunction
```

Comment:
This looks good! See my comments to `FileCreateMessageHarness` and `GsonConverter` also.